src/main/java/net/mcreator/ui/modgui/PlantGUI.java

```diff
@@ -551,6 +559,11 @@ else if (page == 1)
 		plant.generateCondition = generateCondition.getSelectedProcedure();
 		plant.emissiveRendering = emissiveRendering.isSelected();
 
+		useLootTableForDrops.addActionListener(e -> {
+			customDrop.setEnabled(!useLootTableForDrops.isSelected());

```

Comment:
Add this and next line to bottom of openInEditingMode so they get disabled/enabled when loading UI from saved mod lement too